src/core/Akka.Remote/Transport/DotNetty/DotNettyTransport.cs

```diff
@@ -379,13 +378,13 @@ private ServerBootstrap ServerFactory()
             var addressFamily = Settings.DnsUseIpv6 ? AddressFamily.InterNetworkV6 : AddressFamily.InterNetwork;
 
             var server = new ServerBootstrap()
-                .Group(serverEventLoopGroup)
+                .Group(_serverEventLoopGroup)
                 .Option(ChannelOption.SoReuseaddr, Settings.TcpReuseAddr)
                 .Option(ChannelOption.SoKeepalive, Settings.TcpKeepAlive)
                 .Option(ChannelOption.TcpNodelay, Settings.TcpNoDelay)
                 .Option(ChannelOption.AutoRead, false)
                 .Option(ChannelOption.SoBacklog, Settings.Backlog)
-                .Option(ChannelOption.Allocator, Settings.EnableBufferPooling ? (IByteBufferAllocator)new PooledByteBufferAllocator() : new UnpooledByteBufferAllocator())
+                .Option(ChannelOption.Allocator, Settings.EnableBufferPooling ? (IByteBufferAllocator)PooledByteBufferAllocator.Default : UnpooledByteBufferAllocator.Default)
```

Comment:
This would definitely solve the allocator issue for clients... but the server is where we detected this memory leak and IIRC, this `ServerBootstrap` gets re-used to recreate client connections over and over again. Thus, I'm not sure this would actually impact the server. Bleh.